src/InsertionsClient/Api/Providers/InsertionApi.cs

```diff
@@ -127,11 +128,14 @@ public UpdateResults UpdateVersions(string manifestFile, string defaultConfigFil
 
         private void LogStatistics()
         {
-            Trace.WriteLine("Statistics:");
+            StringBuilder stringBuilder = new StringBuilder(900 /* rough size of stats */);
+            stringBuilder.AppendLine("Statistics:");
             foreach (Update update in Enum.GetValues(typeof(Update)).OfType<Update>())
             {
-                Trace.WriteLine($"{update} - {update.GetString()}{Environment.NewLine}{_metrics[update]}");
+                stringBuilder.AppendLine($"{update} - {update.GetString()}{Environment.NewLine}{_metrics[update]}");
             }
+
+            Trace.WriteLine(stringBuilder.ToString());

```

Comment:
Looks like this is going to print an extra newline compared to previous code. Intentional?

Reply:
Thanks for pointing out, I hadn't noticed. Removed now.